libminifi/src/processors/ExtractText.cpp

```diff
@@ -30,20 +31,55 @@
 #include "core/ProcessSession.h"
 #include "core/FlowFile.h"
 
+#if (__cplusplus <= 201103L) || !defined(WIN32)
```

Comment:
I saw an appveyor error related to this, but didn't look into it.

Reply:
Fixed it.
MSVC++ 14 (Visual Studio 2015) claims C++ version to be '98... Some famous quotes of Linus came to my mind when I realised this.